Components/Alert/alert.css

```diff
@@ -0,0 +1,117 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    --simple-alert-bg-color: #bae6fd;
+    --success-alert-bg-color: #d1e7dd;
+    --error-alert-bg-color: #f8d7da;
+    --alert-color: black;
+    --alert-icon-color: black;
+    /* focus color is seen when you use tab key to access buttons or links */
+    --alert-button-focus-outline-color: black;
+}
+
+.alert {
+    width: 32%;
+    max-width: 90%;
+    padding: 0.8em 1.2em;
+    padding-right: 0.6em;
+    background-color: var(--simple-alert-bg-color);
+    border-radius: 0.3em;
+    position: absolute;
+    bottom: 1rem;
+
+    /* flex - properties */
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 1.5em;
+}
+
+.alert.alert-centre {
+    left: 34%;
+}
+
+.alert.alert-right {
+    right: 1rem;
+}
+
+.alert.alert-left {
+    left: 1rem;
+}
+
+@media screen and (max-width: 1030px) {
+    .alert {
+        width: 50%;
+    }
+
+    .alert.alert-centre {
+        left: 25%;
+    }
+}
+
+@media screen and (min-width: 501px) and (max-width: 860px) {
+    .alert {
+        width: 60%;
+    }
+
+    .alert.alert-centre {
+        left: 20%;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .alert {
+        width: 80%;
+    }
+
+    .alert.alert-centre {
+        left: 10%;
+    }
+}
+
+.alert-text {
+    font-size: 1.2rem;
+    color: var(--alert-color);
+    margin: 0;
```

Comment:
you can add this in main css file , so that you don't need to write it again and again

Reply:
Again the same thing, main.css is only for my component library website. Also, i did not make a reset.css file as I was not aware later it was overriding my code and breaking layout. Would do that from next project